pettingzoo/utils/conversions.py

```diff
@@ -294,6 +294,11 @@ def action_space(self, agent):
 
     def reset(self, seed=None, options=None):
         self._observations, self.infos = self.env.reset(seed=seed, options=options)
+
+        # Every environment needs to return infos that contain the same keys as the observations
+        if bool(self.infos) == False:
```

Comment:
I think you can just say if not self.infos or if self.infos = None, unless casting it to a bool is checking something else? But I think it's good to add a blank info in this case like you do, but you should also add a warning that infos is not defined, or something similar.

Also should check something like `isinstance(self.infos, dict) and [ agent not in self.infos.keys() for agent in self.agents ]`, in which case you can do a separate warning that infos is a dict but it does not have the required keys. Then say that we are correcting it basically (my philosophy here is it's not that clear how you're supposed to do it, and if a user is using someone else's environment which isn't properly configured, they should still be able to convert it even if they don't have write access to change the original environment)

Reply:
Updated this to also have a key check, between agent ids and the keys in self.info. There is a warning here as well that the value will be overwritten. I think just an empty dict here is safer than any type of logic to re-use correct key-value pairs and drop incorrect ones. The warning will come up when the conversion is ran, and whoever is converting the environment will be able to address it

Reply:
Fair enough I guess, I was just thinking if they only have an info of `turn: 0` then we could add that. Maybe if it contains none of the agent IDs then we can copy it but if it contains some of them we just say invalid info and override it to be empty?